src/richie/apps/core/templates/menu/language_menu.html

```diff
@@ -0,0 +1,22 @@
+{% load i18n menu_tags cms_tags%}
+{% comment %}
+The name of the languages that appear has to be hardcoded back in because it's automatically translated.
+get_languages will find all available language, including unpublished pages.
+{% endcomment %}
+{% if languages|length > 1 %}
+    {% for language in languages %}
+        {% if language.0 in current_page.get_languages %}
+            {% if lang != language.0 %}
+            <li class="topbar__menu__list__item">
+                <a class="topbar__menu__list__item__link" href="{% page_language_url language.0 %}"
+                title="{% trans "Change to language:" %} {{ language.0 }}">
+                {% if language.0 == "en" %}
```

Comment:
With `{% get_available_languages as languages %}` before the loop, this would be useless, a simple `{{ language.1 }}` will be enough to display untranslated language name as written from `settings.LANGUAGES`

Reply:
Anyway I think it's too long to display the full language name here... some will be longer than french or english!
What do you think about displaying the chooser as: 

EN | **FR** | **DE**   (where english is currently showing)

Reply:
Since we won't use a dropdown menu for now, definitively the short code yes.